scipy/special/_add_newdocs.py

```diff
@@ -9857,11 +9859,78 @@ def add_newdoc(name, doc):
 
     Wrapper for the Cephes [1]_ routine `nbdtr`.
 
+    The negative binomial distribution is also available as
+    `scipy.stats.nbinom`. Using `nbdtr` directly can compare performance
+    compared to the ``cdf`` method of `scipy.stats.nbinom` (see last example).
```

Comment:
Possible typo? Same with the others
```suggestion
    `scipy.stats.nbinom`. Using `nbdtr` directly can improve performance
    compared to the ``cdf`` method of `scipy.stats.nbinom` (see last example).
```

Reply:
Good catch, there were even more of those. Fixed in last commit.